platforms/documentation/docs/src/snippets/bestPractices/content-filtering-do/groovy/settings.gradle

```diff
@@ -0,0 +1,15 @@
+// tag::content-filtering-do[]
+dependencyResolutionManagement {
+    repositories {
+        google {
+            content {
+                // Only use this repository for the androidx.activity group
+                includeGroup "androidx.activity"
```

Comment:
Do we want to give an example of using includeGroupByRegex as that is a very likely scenario for androidx dependencies, as users will want to get *all* AndroidX dependencies from google() repository, not just that single artifact.